exdb/static/exdb/css/ra-home.css

```diff
@@ -0,0 +1,10 @@
+.experience-list , .experience-list td,
+.experience-list th {
+    border: solid black 1px;
+    border-collapse: collapse;
+}
+
+.experience-list tr:not(:first-child):hover {
```

Comment:
I assume from this that we plan to make the rows clickable in the future. If not we should not imply that they are.